xclim/indices/_threshold.py

```diff
@@ -2059,6 +2060,76 @@ def windy_days(
     return out
 
 
+@declare_units(ua="[speed]")
+def jetstream_metric_woolings(
```

Comment:
@tlogan2000 Should this go in atmos or conversion, or an entirely new module, e.g. `synoptic`?

Reply:
I am undecided here.  I don't think `conversion` is really a great fit.   Input/output variables suggest that it should stay in `atmos` but it is definitely quite different from everything else in there.   What other calculations could we imagine in a `synoptic` module  (i.e. is the jetstream calculation going to be the only calculation in that new module)

Reply:
For now it would be, but I can imagine cyclone detection algorithms ending up there.